tooling/noir_js_backend_barretenberg/src/index.ts

```diff
@@ -39,6 +39,7 @@ export class BarretenbergBackend implements Backend {
       await api.srsInitSrs(new RawBuffer(crs.getG1Data()), crs.numPoints, new RawBuffer(crs.getG2Data()));
 
       this.acirComposer = await api.acirNewAcirComposer(subgroupSize);
+      await api.acirInitProvingKey(this.acirComposer, this.acirUncompressedBytecode);
```

Comment:
Can you add a comment here specifying why you are initting the provingKey? This deserves a comment because instantiate is used by the prover and the verifier, so the verifier needing to create a provingKey is not completely intuitive